magento/pwa-studio
# Integrated Webpack HTML Plugin.

## Description

Added Webpack HTML Plugin to generate HTML at compile instead of runtime with UPWARD. HTML will be auto-generated using a template file that carries the boilerplate. In this case `template.html`.

## Related Issue
Closes #1529.

## Verification Steps
1. Compilation should not fail and should generate `index.html` file in the `dist` folder inside `venia-concept`.
2. When you go to any page on the UI and refresh, it should download the exact same `index.html` file irrespective of which page you are in. The previous implementation used to download a different file for different page.

## Screenshots / Screen Captures (if appropriate)
Home Page: 
![image](https://user-images.githubusercontent.com/35203638/63624149-20227580-c5c1-11e9-9657-37e0a8ea4646.png)
Category Page: 
![image](https://user-images.githubusercontent.com/35203638/63624171-32041880-c5c1-11e9-9678-c9be65aa2f4e.png)
Product Page: 
![image](https://user-images.githubusercontent.com/35203638/63624190-447e5200-c5c1-11e9-8188-6188596df178.png)

## Checklist
* Functionality of Venia should not change. This is a compiler modification. A sanity test on the app should be fine.
